LiskHQ/lisk-desktop
# Wallet address should not show legacy address for the new accounts - Closes #3524

### What was the problem?
This PR resolves #3524

### How was it solved?
- Show the full address for legacy and new accounts.
- Show the legacy address only for the legacy accounts.

### How to test
Try with new or legacy account. Click on the truncated address to see full and legacy addresses and then click again to hide them.
Add the following code snippet to `src/utils/api/account/lsk.js`

```
85 +      account.summary.isMigrated = true;
86 +      account.summary.legacyAddress = '1283671237128312313L';
```
for ease of testing.

### How was it tested?
Visually
